components/schema-migrator/migrations/202001221149_add_tenant_storage.up.sql

```diff
@@ -9,4 +9,4 @@ status tenant_status default 'Active'
 );
 
 alter table business_tenant_mappings
-add unique (external_tenant, provider_name);
+add constraint business_tenant_mappings_external_tenant_provider_name_key unique (external_tenant, provider_name);
```

Comment:
why you modified old migration file? if there is a problem you should fix it in the next migration scripts.